lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3918,6 +3918,11 @@ namespace {
                               /*IsCaptureList*/false,
                               Impl.importSourceLoc(decl->getLocation()),
                               name, dc);
+      if (decl->getType().isConstQualified()) {
+        // Note that in C++ there are ways to change the values of const
+        // members, so we don't use WriteImplKind::Immutable storage.
+        result->overwriteSetterAccess(AccessLevel::Private);
```

Comment:
The comment looks out of place to me because the code is setting the setter access level, not the WriteImplKind.

Do you want to add an assertion on WriteImplKind? Or some explanation about why we are setting the access level?

Reply:
Done